starter-code/database.js

```diff
@@ -72,8 +75,8 @@ class Database {
         // Implement the query to delete a user
         // firstName is the name of user that we want to delete
         // remeber once it's finish to comment callback('Error deleting user');
-        
-        callback('Error deleting user');
+        database.collection('users').deleteOne({firstName: firstName},callback);
```

Comment:
this is correct but you can simplify it to:
database.collection(users).deleteOne(**{ firstName }**, callback );